webapp/components/ContentMenu/ContentMenu.vue

```diff
@@ -191,6 +191,15 @@ export default {
               icon: 'user-times',
             })
           }
+          if (this.isAdmin === true) {
+            routes.push({
+              label: this.$t(`settings.deleteUserAccount.name`),
+              callback: () => {
+                this.$emit('delete', this.resource)
+              },
+              icon: 'trash',
+            })
+          }
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-26T14:31:15Z" title="Wednesday, August 26th 2020, 4:31:15 pm +02:00">Aug 26, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -156,6 +156,15 @@ export default {
             path: '/settings',
             icon: 'edit',
           })
+          if (this.isAdmin === true && !this.isOwner) {
+            routes.push({
+              label: this.$t(`settings.deleteUserAccount.name`),
+              callback: () => {
+                this.$emit('delete', this.resource)
+              },
+              icon: 'trash',
+            })
+          }
```


This makes no sense here, because above is the condition that this is the owner.
So we have the overall condition a user which is the owner which is an admin which is not the owner.

Only the next one 👇🏼 is needed …